pgbackrest_backups_retention.py

```diff
@@ -24,33 +24,43 @@
     print("pip install json sys humanize")
     sys.exit(1)
 
-# Retention parameters definition
-incr_backups_retention_time = 7 #days
-full_daily_backup_retention_time = 30 #days
-full_monthly_backup_retention_time = 12 #months
-full_yearly_backup_retention_time = 20 #years
-
 FULL = 'full'
 INCREMENTAL = 'incr'
 
 # Parse command-line arguments
 parser = argparse.ArgumentParser(description='Backup Retention Script')
-parser.add_argument('--stanza', default='db-production-swiss', help='specify the PgBackRest stanza')
-parser.add_argument('--log-file', default='pgbackrest_backups_retention.log', help='specify the log file path')
+parser.add_argument('--stanza', default='db-production-swiss',
```

Comment:
The default stanza shouldn't be set, we should force the user to put his stanza name